ami/main/management/commands/move_project_data.py

```
@@ -0,0 +1,790 @@
+        create_project_name = options.get("create_project")
+        if create_project_name and options.get("target_project"):
+            raise CommandError("Use either --target-project or --create-project, not both")
+
+        if create_project_name:
+            self.log(f"Target project: NEW — '{create_project_name}'")
+            target_project = None
+        elif options.get("target_project"):
+            try:
+                target_project = Project.objects.get(pk=options["target_project"])
+            except Project.DoesNotExist:
+                raise CommandError(f"Target project {options['target_project']} does not exist")
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)